app/src/main/res/values/strings.xml

```diff
@@ -180,4 +180,7 @@
     <string name="manager_in_shop">Manager in shop</string>
     <string name="close">Close</string>
     <string name="see_manager">See Manager</string>
+    <string name="user_in_domain">User in domain</string>
+    <string name="delete">Delete</string>
+    <string name="set_manager">Set Manager</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.set_manager` appears to be unused
 Unused resources make applications larger and slow down builds.